src/pages/home/report/ReportActionItemContentCreated.tsx

```diff
@@ -50,7 +50,7 @@ function ReportActionItemContentCreated({contextValue, parentReportAction, trans
     const {report, action, transactionThreadReport} = contextValue;
 
     const policy = usePolicy(report?.policyID === CONST.POLICY.OWNER_EMAIL_FAKE ? undefined : report?.policyID);
-    const [transaction] = useOnyx(`${ONYXKEYS.COLLECTION.TRANSACTION}${transactionID}`);
+    const [transaction] = useOnyx(`${ONYXKEYS.COLLECTION.TRANSACTION}${transactionID}`, {canBeMissing: false});
```

Comment:
This is [alerting when loading the site](https://expensify.slack.com/archives/C03TQ48KC/p1745496127741719). I think there's some bug here, since it makes no sense to me that we are passing 0 and/or undefined here as transactionID and then here rendering everything as if it was passed a real transaction

Reply:
Yeah that is fair, it was set up to handle it ok so TS is not complaining but I agree ti would be cleaner with more stricter types